### ISSUE TITLE
The user should know where they are

### ISSUE THREAD

**Issue by gotonode** [UniversityOfHelsinkiCS/labtool#945]
The user should be aware of where they are (navigation-wise).

Currently, the main menu doesn't indicate on what page the user is on. It should be easy to implement a color change, but that shouldn't be the only indication.

We're talking about "My page", "Courses" etc. These should change when they are "active".

Also regarding this issue is the Labtool 2.0 logo. Currently it looks deceitfully like a navigation button, but it isn't. Either add a vertical bar on the right side of it to separate it from the navigation buttons (preferred way), or distinguish it in some other way. It could also be a button that links to "My page".

_Additionally_: One minor change, when editing the top menu, is making the left and right top corners square. Currently they are rounded. I think they would look better if they were squared. For clarification, I mean the top left part of Labtool 2.0 logo and the top right part of the Logout-button. Perhaps also add an exit door logo to the Logout-button while we're at it?

**Comment by gotonode** [comment]
I can work on the separation of the logo from the navigation buttons as well as changing the corners of the navigation bar.

Does someone else want to work on making the buttons indicate on which page the user is currently in?

**Comment by gotonode** [comment]
Remove "2.0" from Labtool.

**Comment by gotonode** [comment]
Assign yourself to this issue and to the related pull request if you want to tackle changing the navigation buttons somehow, to indicate on what page the user is currently on.

**Comment by hisahi** [comment]
Done in #1076
